src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeMeasurementSpec.groovy

```diff
@@ -0,0 +1,141 @@
+package life.qbic.datamodel.datasets.datastructure
+
+import life.qbic.datamodel.datasets.Experiment
+import life.qbic.datamodel.datasets.OxfordNanoporeMeasurement
+import life.qbic.datamodel.datasets.datastructure.files.nanopore.Fast5File
+import life.qbic.datamodel.datasets.datastructure.files.nanopore.FastQFile
+import life.qbic.datamodel.datasets.datastructure.folders.DataFolder
+import life.qbic.datamodel.datasets.datastructure.folders.nanopore.*
+import spock.lang.Shared
+import spock.lang.Specification
+
+
+/**
+ * <add class description here>
+ *
+ * @author: Sven Fillinger
+ */
+class OxfordNanoporeMeasurementSpec extends Specification {
+
+    @Shared
+    Fast5PassFolder fast5PassedFolder
+    @Shared
+    Fast5FailFolder fast5FailedFolder
+    @Shared
+    FastQPassFolder fastQPassedFolder
+    @Shared
+    FastQFailFolder fastQFailedFolder
+    @Shared
+    Fast5Folder fast5PooledFolder
+    @Shared
+    FastQFolder fastQPooledFolder
+    @Shared
+    Fast5Folder otherfast5PooledFolder
+    @Shared
+    FastQFolder otherfastQPooledFolder
+    @Shared
+    Map metaData
+
+    def setupSpec() {
+        metaData = [
+                "asic_temp": "32.631687",
+                "device_type" : "promethion",
+                "flow_cell_id": "PAE26306",
+                "flow_cell_product_code": "FLO-PRO002",
+                "flow_cell_position": "2-A3-D3",
```

Comment:
this is not part of the report.md, it is called "device_id" there.
also keep in mind that "protocol" and "started" are part of the other file: "summary"

Reply:
That is a detail only the NanoporeParser will know about. The data model here just expects this property to be there, it does not care from which file it came from :)